### DISCUSSION TITLE
Design Mode

### DISCUSSION THREAD

**Discussion by Romout** [cefsharp/CefSharp#4700] [General]
We have upgraded from 99.2.12 to 119.1.2 and are facing an issue now. We use visual studios integrated WinForms Designer within our own application to enable customers to create dashboards (we call them layouts) using several custom controls as well as a WebBasedControl. This control hosts a CefSharp Browser and allows to configure URL as well as other properties which are provided by a defined javascript within the web page. In the newer version, loading of a page has been disabled.
Is there a way around it? Would it be possible to disable that feature by a switch or something similar?

Thank you in advance for your help!

**Comment by amaitland** [comment]
> In the newer version, loading of a page has been disabled.

The expected behaviour with the `WinForms Designer` is to write the string `ChromiumWebBrowser`.

https://github.com/cefsharp/CefSharp/blob/cefsharp/120/CefSharp.WinForms/ChromiumWebBrowserDesigner.cs#L44

Improvements to design checks were made in https://github.com/cefsharp/CefSharp/issues/4057

> Would it be possible to disable that feature by a switch or something similar?

You'd likely need to submit a `PR` and propose an extension point.
